src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -125,7 +125,10 @@
   </ItemGroup>
   <!-- SocketsHttpHandler implementation -->
   <ItemGroup Condition="'$(TargetsBrowser)' != 'true'">
+    <Compile Include="System\Net\Http\HttpClient.AnyOS.cs" />
```

Comment:
Do we have to include suffix `.AnyOS` here?

However, this should be decided for all affected projects and unified across the repo. I wouldn't want `System.Net.Http` being different and causing confusion just because I personally do not like the suffix.

Reply:
> Do we have to include suffix .AnyOS here?
Yes we do https://github.com/buyaa-n/runtime/blob/a000588324ad46187164c2a8bd69bd36a1d8f841/src/libraries/System.Net.Http/src/System.Net.Http.csproj#L177-L178

> However, this should be decided for all affected projects and unified across the repo. I wouldn't want System.Net.Http being different and causing confusion just because I personally do not like the suffix.

I don't like this suffix too, but I should follow the naming in the project, so for System.Net.Http it is `.AnyOS`